packages/components/src/mobile/bottom-sheet/navigation-screen.native.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import {
```

Comment:
Could you add a README here explaining the what a NavigationScreen is? And how it can be used? What props it can have? Visual screenshots are also appreciated if it makes sense in this context.